### PR TITLE
Development

### PR DESCRIPTION
**Pull request by cristian-ungureanu**

### CODE REVIEW

File: inc/extras.php [hunk 1]
```diff
@@ -69,3 +69,220 @@ function llorix_one_lite_render_title() {
...
+	 *
+	 * @return string
+	 */
+	function llorix_one_lite_pll($input){
+		if(function_exists('pll__')){
+			return pll__($input);
+		}
+		return $input;
+	}
+	add_filter('llorix_one_lite_language_filter','llorix_one_lite_pll');
```
Comment (rodica-andronache) [inline inc/extras.php]: I'm not sure, but shouldn't this add_filter be outsite the if ? Maybe inside an if that checks that the llorix_one_lite_pll exists

File: inc/extras.php [hunk 2]
```diff
@@ -69,3 +69,220 @@ function llorix_one_lite_render_title() {
...
+	function llorix_one_companion_register_string($input,$context){
+		$json_decoded = json_decode($input,true);
+		foreach($json_decoded as $index => $value){
+			foreach ($value as $key => $string){
+				if($key !== 'id' && $string !== 'undefined'){
+					$text = false;
+					if($key === 'text'){
+						$text = true;
+					}
+					pll_register_string('llorix one '.$context, $string, $text);
```
Comment (rodica-andronache) [inline inc/extras.php]: http://prnt.sc/cvd81g when polylang is not installed I think, you need to check if the pll_register_string exists

Actually I tested again, and with polylang installed, I get the same fatal error

File: inc/extras.php [hunk 3]
```diff
@@ -69,3 +69,220 @@ function llorix_one_lite_render_title() {
...
+			array(
+				'icon_value' => 'fa-google-plus-square' ,
+				'link' => '#',
+				'id' => 'llorix_one_lite_56d069b58cb6d'
+			) ) )
+	);
+	llorix_one_companion_register_string($llorix_one_lite_header_social_icons,'header social icons');
+}
+
+include_once( ABSPATH . 'wp-admin/includes/plugin.php' );
```
Comment (rodica-andronache) [inline inc/extras.php]: i'm not sure this is the best way to check if the plugin is active. For example, we check if woocommerce is installed like this: https://github.com/Codeinwp/shop-isle-wporg/blob/6b13ab6abd599ec2ee488613059b1df3b82aecc4/inc/functions/extras.php#L45

I think for polylang, you can check if the class Polylang exists

File: inc/extras.php (hunk 2 above)
Comment (cristian-ungureanu) [inline inc/extras.php]: pll_register_string is inside llorix_one_companion_register_string function which is called only if Polylang class exists. I don't get any fatal errors. Can you please make a print screen?

File: inc/extras.php (hunk 1 above)
Comment (cristian-ungureanu) [inline inc/extras.php]: Yes, I think you're right

File: inc/extras.php (hunk 2 above)
Comment (cristian-ungureanu) [inline inc/extras.php]: Found it

**Comment by rodica-andronache** [comment]
for me, the translations for the repeater it doesn't seem to work.

**Pull request merged**